deploy-jenkins/tasks/main.yml

```diff
@@ -5,7 +5,7 @@
   name: "Update local jenkins-persistent template"
   template:
       src: jenkins-persistent-template.j2
-      dest: /tmp/jenkins-persistent-template.json
+      dest: {{ buildfarm_templates_dir }}/jenkins-persistent-template.json
```

Comment:
A note on this change: We are using `{{ buildfarm_templates_dir }}' when copying all other template files in the repo.
See [here](https://github.com/aerogear/digger-installer/blob/master/android-sdk/tasks/main.yml#L34) and [here](https://github.com/aerogear/digger-installer/blob/master/deploy-nagios/tasks/main.yml#L38).
It is defaulted to `/tmp` in [vars/buildfarm.yml](https://github.com/aerogear/digger-installer/blob/master/vars/buildfarm.yml#L7) and also in rhmap-installer so the logic here isn't changing